src/Lykke.Cqrs/Middleware/Logging/DefaultCommandLogger.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading.Tasks;
+using Common.Log;
+using JetBrains.Annotations;
+using Lykke.Common.Log;
+
+namespace Lykke.Cqrs.Middleware.Logging
+{
+    /// <summary>
+    /// Default command logger.
+    /// </summary>
+    [PublicAPI]
+    public sealed class DefaultCommandLogger : ICommandLogger
+    {
+        private readonly ILog _log;
+
+        /// <summary>C-tor for old logging.</summary>
+        [Obsolete]
+        public DefaultCommandLogger(ILog log)
+        {
+            _log = log;
+        }
+
+        /// <summary>C-tor.</summary>
+        public DefaultCommandLogger(ILogFactory logFactory)
+        {
+            _log = logFactory.CreateLog(this);
+        }
+
+        /// <inheritdoc cref="ICommandLogger"/>
+        public Task<CommandHandlingResult> Log(object handler, object command)
```

Comment:
Return value is not used and it's not actually needed in the logger, so better to change it to the void